releases/point-2011-2-3-6.md

```diff
@@ -0,0 +1,33 @@
+---
+description: This article lists improvements and bug fixes in SysKit Point version 2022.2.3
+--- 
+
+# SysKit Point 2022.2.3
+
+[Schedule a demo](https://www.syskit.com/products/point/request-a-demo/) and [tell us what you think.](https://www.syskit.com/company/contact-us/)
+
+**Product version:** 2022.2.3
+
+**Build number:** 2022.2.3.8
+
+**Release date:** Mar 31, 2022
+
+## Get the Latest Version
+
+* **New to SysKit Point?** Get the latest SysKit Point version from [Azure Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/syskitltd.syskit_point).<br/>
+    Find all steps described in detail in the [deployment article](../installation/deploy-syskit-point.md).
+    
+* **Already using SysKit Point?** Upgrade to the latest SysKit Point version. <br/>
+[![](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fsyskitassetsstorage.blob.core.windows.net%2Fpoint%2FUpdateFilesARM%2FPointUpdateTemplate.json)
+
+See detailed step-by-step instructions on how to upgrade SysKit Point in the [upgrade article](../installation/upgrade-syskit-point.md).
+
+## Improvements & Bug Fixes
+
+* **Improved User privacy options when viewed by Collaborators.** 
+  * **The OneDrive tile and Analytics & Usage tile were removed** from User details for Collaborators. 
+  * When setting up Access, **Admin can now select to give collaborators access to subordinate user's activities.** This option will not be automatically enabled. 
+
+* **Fixed a bug** where archiving Teams resulted in the following error: **Failed to execute Templates backend request ArchiveRequest**.
+
+* **Fixed an issue** where active guest users in Syskit Point were marked as inactive, prompting the admin to delete them.  
```

Comment:
Ovdje je malo složenija situacija ipak. 
Postoji Guest Users Expiration policy gdje možeš definirati kome se šalje mail kada Point otkrije da postoji inactive Guest user. Taj policy mora biti uključen, i tek onda se dešava ovaj prompt; tj. šalje se mail definiranom useru (može biti Manager Guest usera, SysKit Point Administrators ili custom recipient). 
https://sharexuploadsstorage.blob.core.windows.net/sharexuploads/ShareX/2022/03/IGORM_29.03.2022_12.43.38.png
Policy violation task se kreira i onda ta osoba mora odlučiti što sa inactive Guest userom. Uz delete, može reči i Keep.